### PR TITLE
Working on delete functionality

### PR DESCRIPTION
**Pull request by EmmanuelKpendo**

### CODE REVIEW

File: src/components/confirmDelete/ConfirmDelete.jsx [hunk 1]
```diff
@@ -1,41 +1,76 @@
-import './ConfirmDelete.css'
+import "./ConfirmDelete.css";
 import { useParams, Link, useNavigate } from "react-router-dom";
-import React,{useCallback} from 'react';
-import axios from 'axios';
-import { useDispatch, useSelector } from 'react-redux';
-import { deleteInvoice } from '../../invoiceSlice/InvoiceSlice';
+import React, { useCallback } from "react";
+import axios from "axios";
```
Comment (nx-prince-abaidoo) [inline src/components/confirmDelete/ConfirmDelete.jsx:4]: why are you importing the whole axios in the component?

File: src/components/confirmDelete/ConfirmDelete.jsx [hunk 2]
```diff
@@ -1,41 +1,76 @@
...
-  const dispatch = useDispatch()
-  const darkMode = useSelector((state) => state.invoice.isDarkMode)
-
-  const navigate = useNavigate()
+  const navigate = useNavigate();
   const { id } = useParams();
 
-  
+  const deleteInvoice = useCallback(async () => {
+    await axios.delete(`https://invoice-api-9l7b.onrender.com/invoice/${id}`);
```
Comment (nx-prince-abaidoo) [inline src/components/confirmDelete/ConfirmDelete.jsx:16]: avoid repeating endpoint urls in code..
whenthe base url chnages, you will suffer the pain of changing in all components..
we mostly extract the baseURL into cofig file or .env

File: src/components/confirmDelete/ConfirmDelete.jsx [hunk 3]
```diff
@@ -1,41 +1,76 @@
...
-              </p>
-              <div className="cancel-delete-btn">
-                <button className={`confirm-button ${darkMode ? 'cancel-btn-dark' : 'cancel-btn-light'}`} onClick={goBack}>Cancel</button>
-                <button className="confirm-button delete-btn" onClick={()=> dispatch(deleteInvoice(id))}>Delete</button>
-              </div>
-            </div>
+    <section className="delete-modal">
+      <div
+        className={`delete-container ${
+          darkMode ? "delete-container-dark" : "delete-container-light"
```
Comment (nx-prince-abaidoo) [inline src/components/confirmDelete/ConfirmDelete.jsx:37]: i think you need to reconsider how you are implementing your dark Mode..
maybe i need to rather understand what you were trying to do

**Comment by EmmanuelKpendo** [comment]
Kindly explain; I am a bit confused. I don't know if we were supposed to
destructure the methods

**Pull request merged**